src/features/organizations/l10n/messageIds.ts

```diff
@@ -0,0 +1,7 @@
+import { m, makeMessages } from 'core/i18n';
+
+export default makeMessages('feat.organizations', {
+  page: {
+    title: m('Select organizations:'),
```

Comment:
I think this should be `Select organization`.

Reply:
Right!